docs/generated/checks.md

```diff
@@ -10,7 +10,7 @@ KubeLinter includes the following built-in checks:
 
 **Remediation**: Make sure your service's selector correctly matches the labels on one of your deployments.
 
-**Template**: [dangling-service](generated/templates.md#dangling-services)
+**Template**: [dangling-service](templates.md#dangling-services)
```

Comment:
Hmm, we keep it this way because this is what seems to work on https://docs.kubelinter.io/

Reply:
I don't know how docs.kubelinter.io are built but since that's the main spot for docs which I did not know about, I will not fix what's not broken :) Dropped the commit.